profiles/default/linux/x86/17.0/musl/make.defaults

```diff
@@ -1,5 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 CHOST="i686-gentoo-linux-musl"
+RUSTHOST="i686-unknown-linux-musl"
```

Comment:
instead of introducing `RUSTHOST` as a variable into all profiles, couldn't we just rewrite `CHOST` on the fly? In practice it seems to just be about turning all tuples into Debian/Ubuntu ones?

Reply:
sure, but we can remap those host tuples there? having a single mathematical map that translates CHOST/CTARGET to the correct rust values is far preferable to me than RUSTHOST spread all over the profiles, so let's go that way.

Reply:
I don't think I can continue developing this branch.  Someone else can take the PR from here if it is useful.  You can run `grep -FarB1 RUSTHOST profiles` to see which values don't match (all arm and riscv profiles and some of the mips ABIs).